app/src/main/java/io/appium/settings/Settings.java

```diff
@@ -42,6 +44,14 @@ public void onCreate(Bundle savedInstanceState) {
         registerDataConnectionSettingReceiver();
         registerLocationInfoReceiver();
 
+        // https://developer.android.com/about/versions/oreo/background-location-limits
+        if (Build.VERSION.SDK_INT >= 26) {
```

Comment:
We can now use `Build.VERSION_CODES.O` instead of 26

Reply:
changed